examples/sampling/node_classification.py

```diff
@@ -0,0 +1,240 @@
+"""
+If you want a deeper understanding of node classification. You can
+read the example in the  ``examples/pytorch/graphsage/node_classification.py``
+"""
+
+import argparse
+
+import dgl
+import dgl.nn as dglnn
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torchmetrics.functional as MF
+import tqdm
+from dgl.data import AsNodePredDataset
+from dgl.dataloading import (
+    DataLoader,
+    MultiLayerFullNeighborSampler,
+    NeighborSampler,
+)
+from ogb.nodeproppred import DglNodePropPredDataset
+
+
+class SAGE(nn.Module):
+    def __init__(self, in_size, hid_size, out_size):
```

Comment:
hid_size -> hidden_size

There is not benefit to shorten the word, but creates 2 seconds of confusion.